lib/server/services/lhcPeriod/LhcPeriodStatisticsService.js

```diff
@@ -28,15 +28,28 @@ const { NotFoundError } = require('../../errors/NotFoundError');
  * but this breaks limit clause, so pagination.
  * There is related issue for reference: https://github.com/sequelize/sequelize/issues/11983
  */
-const DISTINCT_ENERGIES_IN_STATISTICS_ATTRIBUTE = [
-    `(SELECT GROUP_CONCAT(DISTINCT(r.lhc_beam_energy))
+const DISTINCT_ENERGIES_IN_STATISTICS_ATTRIBUTE = {
+    query: `(SELECT GROUP_CONCAT(DISTINCT(r.lhc_beam_energy))
         FROM runs AS r
         WHERE r.lhc_period_id = lhcPeriod.id 
             AND r.definition = 'PHYSICS'
     )`,
 
-    'distinctEnergies',
-];
+    alias: 'distinctEnergies',
+};
+
+const PDP_BEAM_TYPES_IN_STATISTICS_ATTRIBUTE = {
+    query: `(SELECT r.pdp_beam_type
+        FROM runs AS r
+        WHERE r.lhc_period_id = lhcPeriod.id 
+            AND r.definition = 'PHYSICS'
+            AND r.pdp_beam_type IS NOT NULL
+            AND r.pdp_beam_type != 'cosmic'
```

Comment:
Why? (maybe a link to the definition of what physicists expects? Or a comment on why do you need this)

Reply:
continuing [https://github.com/AliceO2Group/Bookkeeping/pull/1248#discussion_r1405779940] there is one run which is PHYSICS and has `cosmic` beam type and it's not something that should be presented

Reply:
I think you should not handle such a case in here.
Did you noticed RC about this run?

Reply:
Well, I tried achieve the same what was in RCT and it is. If it was considered it's very old.
And if not handle it here, then I need to add some group concats, and introduce changes in API and front, which would bring meaningless information

Reply:
If someone would need this info it might me added later, LHC Periods page is for general information and this is not one

Reply:
changed, it doesn't apply anymore, as problematic run is of bad quality, see https://github.com/AliceO2Group/Bookkeeping/pull/1253